libs/base/ui/overlays/modal/src/modal.component.spec.ts

```diff
@@ -254,6 +254,20 @@ describe('Modal', () => {
     });
   });
 
+  describe('when footer button full width is enabled', () => {
+    beforeEach(async () => {
+      element = await fixture(
+        html`<oryx-modal enableFooter footerButtonFullWidth></oryx-modal>`
+      );
+    });
+
+    it('should add attribute to oryx-button', () => {
+      expect(
+        element.renderRoot.querySelector('oryx-button[full-width]')
+      ).not.toBeNull();
```

Comment:
Better to use `.toBeTruthy()` instead of inversions:
```suggestion
      ).toBeTruthy();
```

Reply:
Now that we don't have that attribute on the oryx-button, we don't need this test anymore.